tests/communication/test_integration.py

```diff
@@ -7,10 +6,7 @@
 import sys
 from semeio.communication import SEMEIOSCRIPT_LOG_FILE
 
-if sys.version_info >= (3, 3):
-    which = shutil.which
-else:
-    which = distutils.spawn.find_executable
+which = shutil.which
```

Comment:
no need to assign the function anymore, just use `shutil.which` where it is supposed to be used.